src/map.h

```diff
@@ -1118,6 +1118,10 @@ class map
          * @return NULL if there is no such field entry at that place.
          */
         field_entry *get_field( const tripoint &p, const field_type_id type );
+        /**
+         * return if field at point is dangerous
```

Comment:
Good thing you documented this. Nobody could have guessed that from the name of the function.\</sarcasm\>

Seriously: if the code is clear enough, you don't need to add a comment for it. Especially when the comment just repeats the function name.